en/app.ftl

```diff
@@ -224,17 +224,26 @@ banner-choose-subdomain-label = Your domain is:
 # $subdomain (url) - User-set subdomain
 banner-choose-subdomain-label = You can make up any address @{ $subdomain }
 
+## Success Messages
+
+# Variables:
+#   $subdomain (url) - User-set subdomain
+success-subdomain-registered = Your domain @{ $subdomain } has been created
+
 ## Error Messages
 
 # Variables:
-#   $number (string) - Maximum number of aliases a user can make on a free account. 
+#   $number (string) - Maximum number of aliases a user can make on a free account.
+#   $unavailable_subdomain (url) - User-set subdomain that is not allowed
```

Comment:
This is a string comment, which only applies to the following string. 

This comment should have added to the new string, with the right variable.

Reply:
Let me file an issue.